app/code/Magento/Backend/Controller/Adminhtml/Index/GlobalSearch.php

```diff
@@ -21,16 +21,26 @@ class GlobalSearch extends \Magento\Backend\Controller\Adminhtml\Index
     protected $_searchModules;
 
     /**
+     * modules that support preview
```

Comment:
Btw this controller should be marked with PHP DocBlock as @ api 
because now it represents extension point for other search modules